src/CoreFx.Private.TestUtilities/src/CoreFx.Private.TestUtilities.csproj

```diff
@@ -29,6 +29,9 @@
     <Compile Include="$(CommonPath)\System\PasteArguments.cs">
       <Link>Common\System\PasteArguments.cs</Link>
     </Compile>
+    <Compile Condition="'$(TargetGroup)' == 'uap'" Include="$(CommonTestPath)\System\Threading\ThreadTestHelpers.cs">
```

Comment:
Basically after moving ThreadTestHelpers, it I'm seeing test projects be able to access RemoteExecutorTestBase but not ThreadTestHelpers